xml/deploy_additional.xml

```diff
@@ -300,6 +300,55 @@
       </listitem>
      </itemizedlist>
     </note>
+    <important>
+     <title>Incorrectly reported image sizes</title>
+     <para>
+      A bug in older kernels caused that an RBD target reported an image size
+      one block (512B) larger than it actually was. Therefore, the LUN could
+      write one additional block of the image. Although this bug is fixed now,
+      existing images can be corrupted because they previously used the block
+      that is not accessible any more. For this reason, the new kernels support
```

Comment:
Not sure about this sentence. It looks correct but might be confusing for users. @ddiss may be you have an idea how it could be changed?

Reply:
Yeah, this seems a bit alarmist, although I'm not exactly sure how to best phrase it...

Corruption should never occur unless `emulate_legacy_capacity` is changed from the kernel default `emulate_legacy_capacity=1` to `emulate_legacy_capacity=0` without growing the image.

I.e. the safe options are:
- for newly created images that have never been exposed via iSCSI, disable `emulate_legacy_capacity` (`emulate_legacy_capacity=0`) during target creation (before it's exposed).
- for images that have been exposed via iSCSI with `emulate_legacy_capacity=1` (SES7+) or any version prior to SES7 (`emulate_legacy_capacity=1` behaviour was always enabled prior to the parameter appearing), there are two options:
  - do nothing; keep the `emulate_legacy_capacity=1` setting as-is
  - disconnect all initiators and take the iSCSI target offline. Grow the RBD image by 512 bytes, then explicitly set `emulate_legacy_capacity=0`. Finally, bring the iSCSI target back online.

The following would be dangerous:
- changing `emulate_legacy_capacity=1` to `emulate_legacy_capacity=0` without growing the image
    - initiators would lose access to the last block of the LUN, which could cause data corruption
- not disconnecting iSCSI initiators when performing the change
    - the `emulate_legacy_capacity=1->0` change and the RBD image 512 byte size increase needs to appear atomic from the SCSI initiator point of view